Annotation/AbstractGraphQLControl.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * @author Thibault Colette <thibaultcolette06@hotmail.fr>
+ * @copyright 2018 Thibault Colette
+ */
+
+namespace Overblog\GraphQLBundle\Annotation;
+
+/**
+ * Annotation for graphql control
+ *
+ * @Annotation
+ * @Target("PROPERTY")
+ */
+abstract class AbstractGraphQLControl
+{
+    /**
+     * Access control access name
+     *
+     * @var string
+     */
+    public $method;
+}
```

Comment:
Can you please give some more :heart:  to [PHP CS](https://github.com/overblog/GraphQLBundle/blob/master/CONTRIBUTING.md#giving-some-love-to-php-cs) :laughing: